OnBoardingCalculator/app/src/main/res/layout/activity_main.xml

```diff
@@ -7,12 +7,194 @@
     tools:context=".MainActivity">
 
     <TextView
+        android:id="@+id/visorPrint"
+        android:layout_width="@dimen/visor_layour_width"
+        android:layout_height="@dimen/visor_layout_height"
+        android:text=""
```

Comment:
this is considered hardcode pls instead use @string/empty_string with "" as value

Reply:
Done! and thanks for the explanation!